atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -137,6 +137,25 @@ private static void addSweepStateTables(Schema schema) {
                 conflictHandler(ConflictHandler.IGNORE_ALL);
             }
         });
+
+        schema.addTableDefinition("sweepAssignedBuckets", new TableDefinition() {
+            {
+                javaTableName("SweepAssignedBuckets");
+                allSafeForLoggingByDefault();
+                rowName();
+                hashFirstNRowComponents(3);
+                rowComponent("shard", ValueType.VAR_LONG);
```

Comment:
Like https://github.com/palantir/atlasdb/pull/7200, I didn't sign the long (although in that PR, there's no intention to have special cells, whereas there is here).

We could reserve e.g. Long.MAX_VALUE instead of signing the long (or can just sign the long and be different to other ASTS but same as previous sweep tables).

Reply:
I don't know if it really matters here, but I think I marginally prefer signed var longs since in the encoding a Long.MAX_VALUE is like a full 9 or 10 bytes